### PR TITLE
feat: support zh_CN translate

### PR DESCRIPTION
**Pull request by lispking**
Closes https://github.com/MystenLabs/walrus-docs/issues/150

### CODE REVIEW

**Comment by mlegner** [comment]
Thanks for the PR. Is there any reason why we should move from `linkcheck` to `linkcheck2`? Also, the linkcheck is not required to build the docs as it is marked as `optional`.

**Comment by lispking** [comment]
Because when linkcheck starts up, the entire service freezes, but switching to linkcheck2 solves the problem.

File: README.md [hunk 1]
```diff
@@ -18,9 +18,19 @@ You can also build and access the documentation locally (assuming you have Rust
 cargo install mdbook
 cargo install mdbook-admonish@1.18.0 --locked
 cargo install mdbook-katex@0.9.0 --locked
+cargo install mdbook-i18n-helpers --locked
 mdbook serve
 ```
 
+### Using translated version
```
Comment (mlegner) [inline README.md:25]: Can you add a link to `TRANSLATING.md` here?

**Review by mlegner — COMMENTED** [review]
Thank you so much for all your work on this, @lispking! I think this setup works well. Maybe @halfprice can skim the translations to see if they make sense.

We will also have to do some follow-up work, which I will add to #150.

File: book.toml [hunk 2]
```diff
@@ -6,7 +6,8 @@ src = "docs"
 title = "Walrus"
 
 [build]
-build-dir = "build"
+# build-dir = "build"
```
[the remark this answers is not in the document]
Comment (lispking) [reply book.toml]: yeah.

**Comment by lispking** [comment]
@mlegner Could you please help take a look at the permission error in the CI? It might require your assistance to handle it.

```shell
/usr/bin/git push --porcelain ***github.com/MystenLabs/walrus-docs.git github-pages-deploy-action/92pjzw8fk:gh-pages
remote: Permission to MystenLabs/walrus-docs.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/MystenLabs/walrus-docs.git/': The requested URL returned error: 403
Changes committed to the gh-pages branch… 📦
Running post deployment cleanup jobs… 🗑️
/usr/bin/git checkout -B github-pages-deploy-action/92pjzw8fk
Reset branch 'github-pages-deploy-action/92pjzw8fk'
/usr/bin/chmod -R +rw github-pages-deploy-action-temp-deployment-folder
/usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
Completed deployment successfully! ✅
Run echo "url=https://mystenlabs.github.io/walrus-docs/pr-preview/pr-154/"  >> $GITHUB_OUTPUT
Run marocchino/sticky-pull-request-comment@331f8f5b4215f0445d3c07b4967662a32a2d3e31
Error: Resource not accessible by integration
```

**Comment by mlegner** [comment]
> @mlegner Could you please help take a look at the permission error in the CI? It might require your assistance to handle it.

We have a workflow to deploy previews of the PRs, which unfortunately doesn't work from forks. This can be ignored.

**Review by halfprice — APPROVED** [review]
I quickly scan through the translation and spot checked a few places. The translation makes sense to me. Just two nits:

- sliver are used as `sliver` or `切片` in different places. May as well to keep it consistent and only use `sliver` everywhere.
- publisher is translated to `发布者` but I feel `发布器` is probably closer to what it is. Also inline with `聚合器`.

**Comment by lispking** [comment]
> I quickly scan through the translation and spot checked a few places. The translation makes sense to me. Just two nits:
> 
> * sliver are used as `sliver` or `切片` in different places. May as well to keep it consistent and only use `sliver` everywhere.
> * publisher is translated to `发布者` but I feel `发布器` is probably closer to what it is. Also inline with `聚合器`.

Thanks for the help from the expert in reviewing. It has been corrected, thank you.

**Pull request merged**